roles/lvm/tasks/volume.yml

```diff
@@ -9,6 +9,15 @@
     - docker
     - bootstrap
 
+- name: update lvg ansible module
+  connection: local
+  run_once: yes
+  get_url:
+    url: https://raw.githubusercontent.com/ansible/ansible-modules-extras/devel/system/lvg.py
```

Comment:
Do we want to pin to a specific commit? Most recent master is [here](https://raw.githubusercontent.com/ansible/ansible-modules-extras/02b68be09dca9760a761d4147f76cfc940c41cba/system/lvg.py).